content/en/docs/examples/microservices-istio/add-new-microservice-version/index.md

```diff
@@ -0,0 +1,166 @@
+        200
+        200
+        ...
+        {{< /text >}}
+
+1.  The previous steps ensure that your new version of `reviews` will work
+    and you can deploy it. You will deploy a single replica of it into
+    production so the real production traffic will start to arrive to your new
+    version. With the current setting, 75% of the traffic will arrive to the old
+    version (three pods of the old version) and 25% will arrive to the new
+    version (a single pod).
+
+    To deploy _reviews v2_, redeploy the new version with the `app=reviews`
+    label, so it will become addressable by the `reviews` service.
+
+    {{< text bash >}}
+    $ kubectl label pods -l version=v2 app=reviews --overwrite
+    pod "reviews-v2-79c8c8c7c5-4p4mn" labeled
+    {{< /text >}}
+
+1.  Now, you access the application web page and observe that the black stars
+    appear for ratings. You can access the page several times and see that
+    sometimes the page is returned with stars (approximately 25% of the time)
+    and sometimes without stars (approximately 75% of the time).
+
+    {{< image width="80%"
+        link="bookinfo-reviews-v2.png"
+        caption="Bookinfo Web Application with black stars as ratings"
+        >}}
+
+1.  If you encounter any problems with the new version in a real-world scenario,
+    you could quickly undeploy the new version, so only the old version will be
+    used:
+
+    {{< text bash >}}
+    $ kubectl delete deployment reviews-v2
+    $ kubectl delete pod -l app=reviews,version=v2
+    deployment "reviews-v2" deleted
+    pod "reviews-v2-79c8c8c7c5-4p4mn" deleted
+    {{< /text >}}
+
+    Access your application's webpage several times and see that now black stars
+    do not appear anymore.
+
+    To put the new version back:
+
+    {{< text bash >}}
+    $ kubectl apply -l app=reviews,version=v2 -f {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo.yaml
+    deployment "reviews-v2" created
+    {{< /text >}}
+
+    Access your application's webpage several times and see that now the black
+    stars are present approximately 25% of the time.
+
+1.  Next, increase the replicas of your new version. You can do it gradually,
+    carefully checking that the number of errors does not increase:
+
+    {{< text bash >}}
+    $ kubectl scale deployment reviews-v2 --replicas=3
+    deployment "reviews-v2" scaled
+    {{< /text >}}
+
+    Now, access your application's webpage several times and see that the black
+    stars appear approximately half the time.
+
+1.  Now, you can decommission the old version:
+
+    {{< text bash >}}
+    $ kubectl delete deployment reviews-v1
+    deployment "reviews-v1" deleted
+    {{< /text >}}
+
+    Accessing the web page of the application will return reviews with black
+    stars only.
+
+In the previous steps, you performed the update of `reviews`. First,
+you deployed the new version without sending it production traffic. You
```

Comment:
Calling it simulated production traffic instead, because we're using Bookinfo as an instructional proxy for what you'd do with a real app, and referring to production/test traffic in that context.